cmd/cvdr/main.go

```diff
@@ -24,24 +25,35 @@ import (
 	"github.com/google/cloud-android-orchestration/pkg/client"
 )
 
-const configPathVar = "CVDR_CONFIG_PATH"
+const (
+	envVarSystemConfigPath = "CVDR_SYSTEM_CONFIG_PATH"
+	// User config values overrieds system config values.
+	envVarUserConfigPath = "CVDR_USER_CONFIG_PATH"
+)
 
-func readConfig(config *cli.Config) error {
-	configPath := os.Getenv(configPathVar)
-	if configPath == "" {
-		// No config file provided
-		return nil
+func loadInitialConfig() (*cli.Config, error) {
+	config := cli.BaseConfig()
+	path, ok := os.LookupEnv(envVarSystemConfigPath)
+	if ok {
+		path = cli.ExpandPath(path)
+		if err := cli.LoadConfigFile(path, config); err != nil {
+			return nil, err
+		}
 	}
-	configFile, err := os.Open(configPath)
-	if err != nil {
-		return fmt.Errorf("Error opening config file: %w", err)
+	path, ok = os.LookupEnv(envVarUserConfigPath)
+	if ok {
```

Comment:
nit: Either restrict `path` and `ok` to the scope of the `if` or use different names for system and user like you had before.

Reply:
true, done.